### PR TITLE
Update joke_hints.py

### PR DESCRIPTION
**Pull request by LatiusAuro**
Added an extra joke hint, playing on how "ADAM" uses the 'But He Sounds Handsome' trope towards Raven Beak.

### CODE REVIEW

File: randovania/exporter/hints/joke_hints.py [hunk 1]
```diff
@@ -29,4 +29,6 @@
     "We now pledge sole allegiance to our liberator, the great one, Dark Samus.",
     "Defeat enemies with the Light Beam to earn Dark Ammo. Defeat enemies with the Dark Beam to earn Light Ammo.",
     "Your Friend Roster is currently empty.",
+    "It is said that reaching <Redacted> without taking any damage will unlock the Metroid Suit.",
```
Comment (henriquegemignani) [inline randovania/exporter/hints/joke_hints.py:32]: I'd skip this one. I'm jaded with people misunderstanding jokes so I don't want to risk.

**Review by henriquegemignani — CHANGES_REQUESTED** [review]

File: randovania/exporter/hints/joke_hints.py [hunk 2]
```diff
@@ -29,4 +29,6 @@
...
+    "Raven Beak? I dont know him, but he sounds hamsome, wouldn't you agree Samus?",
```
Comment (Miepee) [inline randovania/exporter/hints/joke_hints.py:33]: Are you sure that you didn't mean "handsome"?

File: randovania/exporter/hints/joke_hints.py (hunk 1 above)
Comment (LatiusAuro) [reply randovania/exporter/hints/joke_hints.py:32]: Okay then, I'll remove that one

File: randovania/exporter/hints/joke_hints.py (hunk 2 above)
Comment (LatiusAuro) [reply randovania/exporter/hints/joke_hints.py:33]: Ah, yeah, I'll fix that

**Comment by henriquegemignani** [comment]
You don't need to keep updating with main all the time, unless there's a conflict :)

A thing I forgot earlier: can you add a changelog entry?

File: randovania/exporter/hints/joke_hints.py [hunk 3]
```diff
@@ -29,4 +29,5 @@
     "We now pledge sole allegiance to our liberator, the great one, Dark Samus.",
     "Defeat enemies with the Light Beam to earn Dark Ammo. Defeat enemies with the Dark Beam to earn Light Ammo.",
     "Your Friend Roster is currently empty.",
+    "Raven Beak? I dont know him, but he sounds handsome, wouldn't you agree Samus?",
```
Comment (ThanatosGit) [inline randovania/exporter/hints/joke_hints.py:32]: `dont` -> `don't` ? Or are you really allowed to write it like that in english? Never seen it, honestly.

**Review by ThanatosGit — DISMISSED** [review]

File: randovania/exporter/hints/joke_hints.py (hunk 3 above)
Comment (LatiusAuro) [reply randovania/exporter/hints/joke_hints.py:32]: Woops, forgot that my mobile sometimes forgets to do that. I'll fix that now.

**Comment by LatiusAuro** [comment]
I didn't know about the not needing to update it to main, and as for the changelog entry stuff, I'm not 100% sure where I'd need to do that. Technically, the comment for what the push is would what the entry says, but I don't know where else it would be needed.

**Comment by LatiusAuro** [comment]
Hopefully what I did for the changelog entry works after seeing how the other changelog entries work.

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [6.4.0] - 2023-11-??
 
+### Exporter
+- Added: Additional Joke Hint referring to Raven Beak and how he uses the 'But He Sounds Handsome' trope when posing as ADAM.
```
Comment (Miepee) [inline CHANGELOG.md:11]: i'd just shorten this to "Additional Joke Hint referencing Raven Beak"

Also unsure whether it should be in exporter, or in the metroid prime 2 category, as IIRC the joke hints are currently only used for that game

Comment (LatiusAuro) [reply CHANGELOG.md]: Had a look, and the most recent addition to them was under the Prime 2 heading, so moved it there.

**Comment by LatiusAuro** [comment]
Updated the changelog to better fit prior changelogs regarding hints.

**Comment by henriquegemignani** [comment]
I would've been perfectly fine with just a "Added: one additional joke hint.", hah
